vulnerable_people_form/templates/priority-supermarket-deliveries.html

```diff
@@ -15,10 +15,6 @@
         </div>
         <div class=govuk-warning-text>
```

Comment:
i think you want to remove the outer div with the warning text class otherwise the UI will just show an icon with no text

Reply:
oh sorry, you're right